server/chainquery/models/OutputModel.js

```diff
@@ -0,0 +1,77 @@
+const getterMethods = {
+  // Add as needed, prefix all methods with `generated`
+}
+
+export default (sequelize, {
+  STRING, BOOLEAN, INTEGER, TEXT, DECIMAL
+}) => sequelize.define(
+  'output',
+  {
+    id: {
+      primaryKey: true,
+      type: INTEGER,
+      set() { },
+    },
+    transaction_id: {
+      type: INTEGER,
+      set() { },
+    },
+    transaction_hash: {
+      type: STRING,
+      set() { },
+    },
+    value: {
+      type: DECIMAL(18, 8),
+      set() { },
+    },
+    vout: {
+      type: INTEGER,
+      set() { },
+    },
+    type: {
+      type: STRING,
+      set() { },
+    },
+    script_pub_key_asm: {
+      type: TEXT,
+      set() { },
+    },
+    script_pub_key_hex: {
+      type: TEXT,
+      set() { },
+    },
+    required_signatures: {
+      type: INTEGER,
+      set() { },
+    },
+    address_list: {
```

Comment:
This is a JSON array. Not sure if you can specify that here or not.

Reply:
Can't, not supported unless using a virtual field.  I might add a custom getter method.